### PR TITLE
Tidy up file-rbac-extension

### PR DESCRIPTION
**Pull request by YannickWeber**
https://hivemq.kanbanize.com/ctrl_board/22/cards/10399/details/

### CODE REVIEW

File: HEADER [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
\ No newline at end of file
+
+Copyright 2019 dc-square GmbH
```
Comment (hurtadosanti) [inline HEADER:2]: Change it to HiveMQ

File: src/main/java/com/hivemq/extensions/rbac/FileAuthenticatorProvider.java [hunk 2]
```diff
@@ -1,36 +1,36 @@
 /*
- * Copyright 2018 dc-square GmbH
+ *
+ * Copyright 2019 dc-square GmbH
```
Comment (hurtadosanti) [inline src/main/java/com/hivemq/extensions/rbac/FileAuthenticatorProvider.java:3]: Should be HiveMQ

File: src/main/java/com/hivemq/extensions/rbac/configuration/ConfigArchiver.java [hunk 3]
```diff
@@ -1,20 +1,19 @@
 /*
- * Copyright 2018 dc-square GmbH
+ *
+ * Copyright 2019 dc-square GmbH
```
Comment (hurtadosanti) [inline src/main/java/com/hivemq/extensions/rbac/configuration/ConfigArchiver.java:3]: Should be HiveMQ

File: src/main/java/com/hivemq/extensions/rbac/configuration/ConfigCredentialsValidator.java [hunk 4]
```diff
@@ -1,24 +1,28 @@
 /*
- * Copyright 2018 dc-square GmbH
+ *
+ * Copyright 2019 dc-square GmbH
```
Comment (hurtadosanti) [inline src/main/java/com/hivemq/extensions/rbac/configuration/ConfigCredentialsValidator.java:3]: Should be HiveMQ

File: src/main/java/com/hivemq/extensions/rbac/configuration/ConfigParser.java [hunk 5]
```diff
@@ -1,26 +1,25 @@
 /*
- * Copyright 2018 dc-square GmbH
+ *
+ * Copyright 2019 dc-square GmbH
```
Comment (hurtadosanti) [inline src/main/java/com/hivemq/extensions/rbac/configuration/ConfigParser.java:3]: Should be HiveMQ

File: src/main/java/com/hivemq/extensions/rbac/configuration/Configuration.java [hunk 6]
```diff
@@ -1,35 +1,38 @@
 /*
- * Copyright 2018 dc-square GmbH
+ *
+ * Copyright 2019 dc-square GmbH
```
Comment (hurtadosanti) [inline src/main/java/com/hivemq/extensions/rbac/configuration/Configuration.java:3]: Should be HiveMQ

File: src/main/java/com/hivemq/extensions/rbac/configuration/ExtensionConfiguration.java [hunk 7]
```diff
@@ -1,24 +1,23 @@
 /*
- * Copyright 2018 dc-square GmbH
+ *
+ * Copyright 2019 dc-square GmbH
```
Comment (hurtadosanti) [inline src/main/java/com/hivemq/extensions/rbac/configuration/ExtensionConfiguration.java:3]: Should be HiveMQ

File: src/main/java/com/hivemq/extensions/rbac/configuration/NotMarshallableException.java [hunk 8]
```diff
@@ -1,20 +1,19 @@
 /*
- * Copyright 2018 dc-square GmbH
+ *
+ * Copyright 2019 dc-square GmbH
```
Comment (hurtadosanti) [inline src/main/java/com/hivemq/extensions/rbac/configuration/NotMarshallableException.java:3]: Should be HiveMQ

File: src/main/java/com/hivemq/extensions/rbac/configuration/XmlParser.java [hunk 9]
```diff
@@ -1,27 +1,26 @@
 /*
- * Copyright 2018 dc-square GmbH
+ *
+ * Copyright 2019 dc-square GmbH
```
Comment (hurtadosanti) [inline src/main/java/com/hivemq/extensions/rbac/configuration/XmlParser.java:3]: Should be HiveMQ

File: src/main/java/com/hivemq/extensions/rbac/configuration/entities/ExtensionConfig.java [hunk 10]
```diff
@@ -1,26 +1,30 @@
 /*
- * Copyright 2018 dc-square GmbH
+ *
+ * Copyright 2019 dc-square GmbH
```
Comment (hurtadosanti) [inline src/main/java/com/hivemq/extensions/rbac/configuration/entities/ExtensionConfig.java:3]: Should be HiveMQ

File: src/main/java/com/hivemq/extensions/rbac/configuration/entities/PasswordType.java [hunk 11]
```diff
@@ -1,20 +1,19 @@
 /*
- * Copyright 2018 dc-square GmbH
+ *
+ * Copyright 2019 dc-square GmbH
```
Comment (hurtadosanti) [inline src/main/java/com/hivemq/extensions/rbac/configuration/entities/PasswordType.java:3]: Should be HiveMQ

File: src/main/java/com/hivemq/extensions/rbac/configuration/entities/Permission.java [hunk 12]
```diff
@@ -1,20 +1,19 @@
 /*
- * Copyright 2018 dc-square GmbH
+ *
+ * Copyright 2019 dc-square GmbH
```
Comment (hurtadosanti) [inline src/main/java/com/hivemq/extensions/rbac/configuration/entities/Permission.java:3]: Should be HiveMQ

File: src/test/resources/test-extension-config.xml [hunk 13]
```diff
@@ -1,4 +1,22 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+
+
+    Copyright 2019 dc-square GmbH
```
Comment (hurtadosanti) [inline src/test/resources/test-extension-config.xml:5]: Should be HiveMQ

**Review by hurtadosanti — COMMENTED** [review]
I think a replace all will be better for the dc-square licence comment

File: gradle/wrapper/gradle-wrapper.properties [hunk 14]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-bin.zip
```
[the remark this answers is not in the document]
Comment (YannickWeber) [reply gradle/wrapper/gradle-wrapper.properties:3]: 7.4 points to the latest 7.4.x its also what `gradle init` generates.

**Review by hurtadosanti — APPROVED** [review]

**Pull request merged**